admin_manual/configuration_user/two_factor-auth.rst

```diff
@@ -76,3 +83,14 @@ The associations of removed providers can be cleaned up via :ref:`occ <occ>`::
  sudo -u www-data php occ twofactorauth:cleanup <provider_id>
 
 .. warning:: This operation is irreversible. Only run it for providers you do not intend to enable again.
```

Comment:
Sound a bit like you can never use it again. It's just the current settings are all lost, and if you enable it you have to set up everything again (might be interesting for people who have messed up something with their setup)?